components/core/sourceCodeLink.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Link from 'components/core/link'
+import Div from 'components/core/div'
+import styled from 'styled-components'
+
+const SourceCodeLink = styled(Link)`
+  margin-left: ${props => props.theme.spaces.small}px;
+`
+
+const SourceCodeLinkContainer = styled(Div)`
+  align-items: center;
+`
+
+const GithubLogo = styled.img`
+  width: 20px;
+  height: 50px;
+`
+
+export default () => (
+  <SourceCodeLinkContainer>
```

Comment:
you can wrap the whole thing in an `<a>` to make everything (including the icon) a link.